packages/engine/src/initializeEngine.ts

```diff
@@ -160,6 +155,11 @@ export const initializeCoreSystems = async (injectedSystems?: SystemModuleType<a
       uuid: 'xre.engine.AssetSystem',
       type: SystemUpdateType.FIXED_LATE,
       systemLoader: () => import('./scene/systems/AssetSystem')
+    },
+    {
+      uuid: 'xre.engine.OutgoingActionSystem',
```

Comment:
moved the wrong system... OutgoingActionSystem needs to stay where it was

Reply:
Ah, that was a rebasing error, reverting.

Reply:
Reverted